CRM/Pcadets/Page/PowerTheCadets.php

```diff
@@ -89,12 +90,15 @@ public function run() {
         // If remain_anonymous field is not check, store name and amount in the sponsors
         if (!$remainAnonymous) {
         $sponsors[] = (!empty($contribution["custom_{$displayNameCustomFieldId}"]) ? $contribution["custom_{$displayNameCustomFieldId}"] : CRM_Contact_BAO_Contact::displayName($contribution['contact_id']));
+        } else {
+            $sponsors[] = "Anonymous";
         }
       }
       $sponsors = array_unique($sponsors);
 
       // Get the totalContribution percentage base on the total_meal_value_per_day
-      $percentage = (int) $totalContribution / $totalMealValuePerDay * 100;
+      $percentageRaw = (int) $totalContribution / $totalMealValuePerDay * 100;
+      $percentage = ceil($percentageRaw / 10) * 10;
```

Comment:
@JustinDMoore Curious what this line does? Rounding the percentage to the nearest 10, I think?